examples/mysql-galera/README.md

```diff
@@ -0,0 +1,171 @@
+
+### Create services and controllers for the remaining nodes
+
+Repeat the same previous steps for ```pxc-node2``` and ```pxc-node3```
+
+When complete, you should be able connect with a mysql client to the IP address
+ service ```pxc-cluster``` to find a working cluster
+
+### Confirming replication controller (or pods) and services are running
+
+Shown below are examples of Using ```kubectl``` from within the ```./examples/mysql-galera``` directory, the status of the lauched replication controllers and services can be confirmed
+
+```
+$ kubectl create -f examples/mysql-galera/pxc-cluster-service.yaml 
+services/pxc-cluster
+
+$ kubectl create -f examples/mysql-galera/pxc-node1.yaml 
+services/pxc-node1
+replicationcontrollers/pxc-node1
+
+$ kubectl create -f examples/mysql-galera/pxc-node2.yaml 
+services/pxc-node2
+replicationcontrollers/pxc-node2
+
+$ kubectl create -f examples/mysql-galera/pxc-node3.yaml 
+services/pxc-node3
+replicationcontrollers/pxc-node3
+
+```
+
+Verify everything is running:
+
+```
+$ kubectl get rc,pods,services
+CONTROLLER   CONTAINER(S)   IMAGE(S)                                    SELECTOR           REPLICAS
+pxc-node1    pxc-node1      capttofu/percona_xtradb_cluster_5_6:beta    name=pxc-node1     1
+pxc-node2    pxc-node2      capttofu/percona_xtradb_cluster_5_6:beta    name=pxc-node2     1
+pxc-node3    pxc-node3      capttofu/percona_xtradb_cluster_5_6:beta    name=pxc-node3     1
+NAME              READY     STATUS    RESTARTS   AGE
+pxc-node1-h6fqr   1/1       Running   0          41m
+pxc-node2-sfqm6   1/1       Running   0          41m
+pxc-node3-017b3   1/1       Running   0          40m
+NAME          LABELS    SELECTOR           IP(S)            PORT(S)
+pxc-cluster   <none>    unit=pxc-cluster   10.100.179.58    3306/TCP
+pxc-node1     <none>    name=pxc-node1     10.100.217.202   3306/TCP
+                                                            4444/TCP
+                                                            4567/TCP
+                                                            4568/TCP
+pxc-node2     <none>    name=pxc-node2     10.100.47.212    3306/TCP
+                                                            4444/TCP
+                                                            4567/TCP
+                                                            4568/TCP
+pxc-node3     <none>    name=pxc-node3     10.100.200.14    3306/TCP
+                                                            4444/TCP
+                                                            4567/TCP
+                                                            4568/TCP
+
+```
+
+The cluster should be ready for use!
+
+### Connecting to the cluster
+
+Using the IP address for the ```pxc-cluster``` service, and running interactively the capttofu/percona_xtradb_cluster_5_6 image for the mysql client on one of the running containers of the cluster, one can connect to the cluster and verify the cluster size (should be 3):
```

Comment:
what if you do something like `kubectl exec pxc-node1 -i -t bash`? Same problem?